static/style.css

```diff
@@ -1,20 +1,119 @@
-body            { font-family: sans-serif; background: #eee; }
-a, h1, h2       { color: #377ba8; }
-h1, h2          { font-family: 'Georgia', serif; margin: 0; }
-h1              { border-bottom: 2px solid #eee; }
-h2              { font-size: 1.2em; }
-form			{ display: inline; }
-div				{ font-family: sans-serif;	background: #eee}
-
-.page           { margin: 2em auto; width: 35em; border: 5px solid #ccc;
-                  padding: 0.8em; background: white; }
-.entries        { list-style: none; margin: 0; padding: 0; }
-.entries li     { margin: 0.8em 1.2em; }
-.entries li h2  { margin-left: -1em; }
-.add-entry      { font-size: 0.9em; border-bottom: 1px solid #ccc; }
-.add-entry dl   { font-weight: bold; }
-.metanav        { text-align: right; font-size: 0.8em; padding: 0.3em;
-                  margin-bottom: 1em; background: #fafafa; }
-.flash          { background: #cee5F5; padding: 0.5em;
-                  border: 1px solid #aacbe2; }
-.error          { background: #f0d6d6; padding: 0.5em; }
\ No newline at end of file
+body { 
+	font-family: sans-serif;
+	background: #eee;
+}
+
+a, h1, h2 {
+	color: #377ba8;
+}
+
+h1, h2 {
+	font-family: 'Georgia', serif;
+	margin: 0;
+}
+
+h1 {
+	border-bottom: 2px solid #eee;
+}
+
+h2 {
+	font-size: 1.2em;
+}
+
+form {
+	display: inline;
+}
+
+div {
+	font-family: sans-serif;
+	background: #eee
+}
+
+input[type="text"] {
+	width: 100%;
+}
+
+textarea {
+	width: 100%;
+}
+
+input[type="submit"], input[type="submit"]:focus, input[type="submit"]:active {
+    /* Remove all decorations to look like normal text */
+    height: 25px;
```

Comment:
The indentation is inconsistent, here there are 4 spaces, above - 8. I usually prefer using 4 spaces everywhere (and in js files too).